airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/IntegrationTest.kt

```diff
@@ -313,8 +313,11 @@ abstract class IntegrationTest(
         fun isNamespaceOld(namespace: String, retentionDays: Long = 30): Boolean {
             val cleanupCutoffDate = LocalDate.now().minusDays(retentionDays)
             val matchResult = randomizedNamespaceRegex.find(namespace)
+            if (matchResult == null || matchResult.groups.isEmpty()) {
+                return false
+            }
```

Comment:
this is just a safety thing - nothing is broken afaik, just seemed reasonable to add this check